OpenRA.Mods.Common/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -241,7 +241,8 @@ public override bool HandleKeyPress(KeyInput e)
 			{
 				var key = Hotkey.FromKeyInput(e);
 
-				if (key == Game.Settings.Keys.PauseKey && World.LocalPlayer != null) // Disable pausing for spectators
+				if (key == Game.Settings.Keys.PauseKey && World.LocalPlayer != null &&
+					(Game.IsHost || player.WinState != WinState.Lost)) // Disable pausing for spectators and defeated players
```

Comment:
Actually this isn't quite right.  If a real player loses the game, and then as a spectator changes to a different player's view, then `player.WinState` will be for the observed player and they will be allowed to pause.